Mergin/monthly_contributors_error_dialog.py

```diff
@@ -0,0 +1,40 @@
+import os
+from qgis.PyQt import uic
+from qgis.PyQt.QtWidgets import QDialog
+from qgis.PyQt.QtCore import QUrl
+from qgis.PyQt.QtGui import QDesktopServices
+
+ui_file = os.path.join(os.path.dirname(os.path.realpath(__file__)), "ui", "ui_monthly_contributors_error_dialog.ui")
+
+
+class MonthlyContributorsErrorDialog(QDialog):
+    def __init__(self, e, parent=None):
+        QDialog.__init__(self, parent)
+        self.ui = uic.loadUi(ui_file, self)
+
+        self.server_response = e.server_response
+        self.set_dialog_style()
+
+        self.cancel_btn.clicked.connect(self.reject)
```

Comment:
Just for your information when you want only standard action there is a [QDialogButtonBox](https://doc.qt.io/qt-5/qdialogbuttonbox.html) to layout out standard button 

But you have less fine grained support to lay them as you want